test/unit/parse-test.js

```diff
@@ -104,7 +84,7 @@ describe('parse()', () => {
     it('produces media type', () => assert.match(mediaType, reMediaType));
     it('the parse result contains annotation elements', () => assert.isFalse(apiElements.annotations ? apiElements.annotations.isEmpty : undefined));
     it('the annotations are warnings', () => assert.equal(apiElements.warnings ? apiElements.warnings.length : undefined, apiElements.annotations.length));
-    it('the first warning is about falling back to API Blueprint', () => assert.include(apiElements.warnings.getValue(0), 'to API Blueprint'));
+    it('the first warning is about falling back to API Blueprint', () => assert.include(apiElements.warnings.getValue(0), 'assuming API Blueprint'));
```

Comment:
Perhaps nicer to make this multi-line?

```suggestion
    it('the first warning is about falling back to API Blueprint', () => {
      assert.include(apiElements.warnings.getValue(0), 'assuming API Blueprint');
    });
```

Maybe for separate change for whole code-base though.

Reply:
That's exactly what I'm doing for an hour already 😄